### DISCUSSION TITLE
Add a flag to disable protections outside of claimed areas...

### DISCUSSION THREAD

**Discussion by zagarjr** [GriefPrevention/GriefPrevention#1354] [Ideas]
# Summary
**As** a player
**I want** to build / destroy / exploit all the things... unless someone has claimed it.
**Because**, if nobody owns it, why is it forbidden to alter it?

Optional questions:

**Is your feature request related to a problem? Please describe.**

TL;DR - GP's aggressive protection of unclaimed areas gets in the way of 5yo fun, so I'd like to be able to turn that off.

My 5yo daughter and her friends play minecraft in creative mode.  As you can imagine, kinder-aged kids lack the discipline to not wreck other people's stuff.  Grief Prevention is a little difficult for a 5yo to understand, but I could deal with that if it *only* prevented grief within their own "sandbox".

**Describe alternatives you've considered**

Command blocks *can* be used to put invisible falling_blocks on players' heads based on target selector arguments (e.g. @a[distance=100]).  Once the block is stuck on your head, you can't use tools or punch things.  While this works, for some definitions of work, it requires a level of programming I don't current have.

**Comment by Jikoo** [comment]
Sounds like you're probably using `creative` claims mode - try `survival` instead.
